app/partials/app.cjsx

```diff
@@ -72,7 +70,7 @@ module.exports = React.createClass
   getEnv: ->
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env || 'staging'
+    @state?.env || browser_env[1] || 'staging'
```

Comment:
It's basically copied from the Panoptes client here: https://github.com/zooniverse/panoptes-javascript-client/blob/master/lib/config.js#L76